docs/source/utils.rst

```diff
@@ -7,6 +7,30 @@ A simple utility to import something by its string name.
 
 .. autofunction:: import_item
 
+A way to expand the signature of the ``HasTraits`` class constructor, this enables auto-completion of trait-names in IPython and xeus-python when having Jedi>=0.15.
```

Comment:
I'd update the docs here to reflect the decorator name change to say that we're adding trait names to the signature.